posthog/hogql/visitor.py

```diff
@@ -153,8 +153,9 @@ def visit_select_query_type(self, node: ast.SelectQueryType):
             self.visit(expr)
         for expr in node.aliases.values():
             self.visit(expr)
-        for expr in node.columns.values():
-            self.visit(expr)
+        # FIXME: This causes an exponential explosion when traversing the tree.
+        # for expr in node.columns.values():
+        #     self.visit(expr)
```

Comment:
Couldn't make out the root cause. Columns and aliases seem to be duplicated.

<img width="1117" alt="Screenshot 2024-01-30 at 18 28 14" src="https://github.com/PostHog/posthog/assets/1851359/ea40a51d-41f4-4fca-945c-627b9956b487">